libs/challenge-registry/ui/src/lib/challenge-card/_challenge-card-theme.scss

```diff
@@ -98,7 +98,18 @@
     color: #000;
 
     &.active {
-      // wait until the colors are determined
+      border-color: #71c663 !important;
```

Comment:
I read in the past that using `!important` should be avoided. I did a quick search to refresh my memory of [why](https://www.w3schools.com/css/css_important.asp).

> Tip: It is good to know about the `!important` rule, you might see it in some CSS source code. However, do not use it unless you absolutely have to.

I'd say that it's OK to use `!important` in the style or theme of a component like done here - though maybe this can be avoided using more specific CSS selectors? - but not in a reusable styles and themes like in the project `challenge-registry-styles` and `challenge-registry-themes`.

Reply:
If I remember correctly, the order of priority of CSS styles is defined by the order the .css files are imported in `index.html` (we have no control over this). I think that another factor is the order of the classes specified to an element in the .html file. Is that actually true? If yes, you could create a custom CSS class and list it first/last on the element.

>  I think it's fine to use !important on challenge-card ?

Yes!

Reply:
>  I think that another factor is the order of the classes specified to an element in the .html file. Is that actually true? If yes, you could create a custom CSS class and list it first/last on the element.
 
Yes, it is true. You remind me of the order and the issue is because the theme (`_challenge-card-theme.scss`) is loaded before the `challenge-card.component.scss` where it has:
```scss
.challenge-card-status-tag {
  border: 1px solid;
}
```
If `border` is used and no color provided, it will use the default color, which will overwrite the border color we define in the `_challenge-card-theme.scss` since it's called later. Splitting `border` into `border-width` and `border-style` can avoid overwrite the border color. I will make the changes.